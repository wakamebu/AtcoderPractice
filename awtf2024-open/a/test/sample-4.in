20 6
0 3441380 120768398 229897071 231209282 232046760 254924545 325399248 385631087 400098966 480503302 501372095 502644652 524585010 541761042 691400171 725009462 767549897 837806226 927396743
