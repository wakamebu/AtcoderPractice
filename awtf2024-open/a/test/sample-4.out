135453315.33333333333333333333
